lib/auth/tls_test.go

```diff
@@ -1720,6 +1720,32 @@ func TestGetCertAuthority(t *testing.T) {
 	}, true)
 	require.True(t, trace.IsAccessDenied(err))
 
+	// generate server keys for proxy
+	proxyClt, err := tt.server.NewClient(TestIdentity{I: BuiltinRole{Username: "00000000-0000-0000-0000-000000000001", Role: types.RoleProxy}})
```

Comment:
Suggestion: formatting:

```suggestion
	proxyClt, err := tt.server.NewClient(TestIdentity{
		I: BuiltinRole{
			Username: "00000000-0000-0000-0000-000000000001",
			Role: types.RoleProxy,
		},
	})
```